python/paddle/trainer/config_parser.py

```diff
@@ -1705,6 +1722,22 @@ def __init__(
             self.set_layer_size(block_expand_conf.block_x * block_expand_conf.block_y
                 * block_expand_conf.channels)
 
```

Comment:
maxout is like an activation, better check the activation of the conv before it is linear activated.